requirements.in

```diff
@@ -14,6 +14,7 @@ django-parler
 django-ilmoitin
 factory-boy
 graphene-django
+graphene-file-upload
```

Comment:
I think, alphabetically it should come after `graphene-federation` 😅

Reply:
😭 so close yet so far, I thought I had gotten it right.